src/PluralResolver.js

```diff
@@ -111,7 +112,18 @@ class PluralResolver {
   getRule(code, options = {}) {
     if (this.shouldUseIntlApi()) {
       try {
-        return new Intl.PluralRules(getCleanedCode(code === 'dev' ? 'en' : code), { type: options.ordinal ? 'ordinal' : 'cardinal' });
+        const locales = getCleanedCode(code === 'dev' ? 'en' : code);
```

Comment:
Tiny nit: Instead of `const locales`, I think naming this `const cleanedCode` would be a bit clearer, since the output is still a single (not-plural) code for a locale